src/MeetingsSDKAdapter/controls/AudioControl.test.js

```diff
@@ -0,0 +1,71 @@
+import MeetingsSDKAdapter from '../../MeetingsSDKAdapter';
+import createMockSDK from '../../mockSdk';
+
+describe('Audio Control', () => {
+  let meeting;
+  let meetingID;
+  let meetingsSDKAdapter;
+  let mockSDK;
+
+  beforeEach(() => {
+    mockSDK = createMockSDK();
+    meetingID = 'meetingID';
+    meetingsSDKAdapter = new MeetingsSDKAdapter(mockSDK);
```

Comment:
please use the `createTestMeetingsSDKAdapter()` function to prevent code duplication